corehq/apps/hqcase/utils.py

```diff
@@ -255,3 +256,11 @@ def bulk_update_cases(domain, case_changes, device_id):
         case_block = ElementTree.tostring(case_block.as_xml())
         case_blocks.append(case_block)
     return submit_case_blocks(case_blocks, domain, device_id=device_id)
+
+
+def resave_case(domain, case, send_post_save_signal=True):
+    from corehq.form_processor.change_publishers import publish_case_saved
+    if should_use_sql_backend(domain):
+        publish_case_saved(case, send_post_save_signal)
+    else:
+        CommCareCase.get_db().save_doc(case._doc)  # don't just call save to avoid signals
```

Comment:
Based on the comment on this line if `send_post_save_signal == True` then I think this should call `case.save()`

Reply:
Thanks for catching that. Updated.